.travis.yml

```diff
@@ -39,8 +39,6 @@ branches:
 env:
   global:
     - NPM_CONFIG_PROGRESS="false"
-    - secure: nvLqQdU2CGwvsQ3E1M/6HgBJAfmSzd90bDYQ1vkAkwGj+VOVQauqj7RDJACJS2XysJjzIvzNUOtKFvTSzThye167sKAZnQMmj9U8ATdlJfv5xq88QkkoCuJIOPtVnOWS8gBrDAy9d+Xwi22ZcE5kAAov1gNDmnX6YcPwW8y09vD+Jb1WkxofwTR5U3dP7pxVNRwgXY12g2lmbj+TZy5RHvjs1/ESSVtt/5Wq+Ubv9O6KAVPrh/VdVTYyvgynvwzz5jG3qe7981W8rUNIt+ihzdxJMSaoQkiBs5UM3qi8ibqKI2KHdDmGQg+vc/c5mjCCd9FAz7lF+fJJQ3YThTLmIN3u7Hil+o92OaAt2knwtZ3HRv/jbVQfnU0L9hqHw/kBYE37O4x4/ToJyR3EgbvFXqgiVph0eDUnsmi8PZc5FUiwb8r85thQmfUB+urkOXbnOyHQ9B74T+UjAvlaqLdQ53/9UTzgvLNm2z6RU65G9QYJVMSzj82K2rZ9AyVwjZMp/iuQDoUcUsuMhKDh2genHnxRtQa/TbLQM05oYHRqFa9oT3Q7C7QSSL8mmi/QMfGlTfxhM49CZsbcU2qbMCMCL7ZGWaSiXiEy7XICOzJnOiKFowDK5ihAZaQpRNquMwUSIMnnqrFgfWDbQSUmkOAwSm+2u2+SsXhT+WQfraRfSSI=
```

Comment:
These were moved elsewhere right? Is it okay that these were publicly visible on github?